frontend/src/components/ResultsTable.js

```diff
@@ -14,8 +16,15 @@ class ResultsTable extends Component {
         switch (typeof v) {
           case 'boolean':
             return v.toString();
+          case 'string':
+            // assume any multiline string is code
+            if (v.indexOf('\n') > 1) {
```

Comment:
Should this be `> -1`? The way it is a multiline file with a blank line will not be detected.